charts/telemetry-server/values.yaml

```diff
@@ -130,19 +134,32 @@ config:
     port: 9999
   dbs:
     telemetry:
-      driver: sqlite3
-      params: /var/lib/tsvc/data/telemetry.db
+      driver: postgres
+      params: "<TELEMETRY_DB_CONNECTION_STRING>"
     operational:
       driver: postgres
-      params: "postgres://postgres:telemetry@postgres:5432/operational"
+      params: "<OPERATIONAL_DB_CONNECTION_STRING>"
     staging:
-      driver: sqlite3
-      params: /var/lib/tsvc/data/staging.db
+      driver: postgres
+      params: "<STAGING_DB_CONNECTION_STRING>"
   logging:
-    level: info
+    level: debug
     location: stderr
     style: text
   # This is for testing purposes only
   auth:
     secret: VGVzdGluZ1NlY3JldAo=
     duration: 1w
+
+# TODO: Transfer responsability of initializing Postgres instance to CI/CD pipeline
+rdsInit:
+  image:
+    repository: postgres
+    tag: 16
+  postgres:
+    host: <POSTGRES_HOST>
```

Comment:
Consistency comment - above you wrapped <...> placeholders with quotes, but here you don't have any quotes.